### PR TITLE
Added new Project : Water Reminder

### LINKED ISSUE #270: New Project : Water Reminder
## 🚀 Feature

Water reminder program using python.

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes, I have read the contributing guidelines.

## Motivation
Nowadays we remain so occupied with the day-to-day work that we miss the essential things; like drinking water and keeping ourselves hydrated.
## Pitch

A simple python project which reminds the user to drink water after predefined intervals

### PR DESCRIPTION
**Pull request by LiQuiD-404**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes, I have read the Contributing guidelines.

### Description

A simple Water Reminder Program built using Python which will help us reach our daily water consumption goal.

Fixes #270 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

File: Desktop Application/Basic/Python/Water Reminder/Readme.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+
+
+
+   
+ 
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Water Reminder/Readme.md:6]: Remove the unwanted space form here !

File: Desktop Application/Basic/Python/Water Reminder/Readme.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  <h3 align="center">Hydrate - Water Reminder</h3>
+
+  <p align="center">
+  A simple Python Program that will help you reach your daily water consumption goal
+  </div>
+   
+  
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Water Reminder/Readme.md:14]: Remove these lines !

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@LiQuiD-404 Remove all extra space from every files and add gif or photos in `readme.md` file.

File: Desktop Application/Basic/Python/Water Reminder/Readme.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Contact
+
+Shubham Singh - [LinkedIn](https://www.linkedin.com/in/shubham-singh-519769220/) 
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Water Reminder/Readme.md:87]: Remove the space !

File: Desktop Application/Basic/Python/Water Reminder/Readme.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+
+
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Water Reminder/Readme.md:91]: Remove all white space

File: Desktop Application/Basic/Python/Water Reminder/Readme.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+### Built With
+
+* [Python](https://www.python.org/)
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Getting Started
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Water Reminder/Readme.md:42]: Remove white space

File: Desktop Application/Basic/Python/Water Reminder/Readme.md (hunk 2 above)
Comment (LiQuiD-404) [reply Desktop Application/Basic/Python/Water Reminder/Readme.md]: Made the necessary changes. Kindly review.

**Review by Kushal997-das — COMMENTED** [review]
Remove all white space

**Comment by LiQuiD-404** [comment]
Removed all the extra whitespaces. Kindly review.

**Review by Kushal997-das — APPROVED** [review]

**Pull request merged**